### ISSUE TITLE
There is no such thing as a "frequency card" for System Entrance/Destination Pipes

### ISSUE THREAD

**Issue by aaronhowser1** [RS485/LogisticsPipes#1444]
![image](https://user-images.githubusercontent.com/15698722/81041913-9677b700-8e74-11ea-9128-5fbacff4ac0c.png)
![image](https://user-images.githubusercontent.com/15698722/81041916-9aa3d480-8e74-11ea-8898-9be9896c42ef.png)
![image](https://user-images.githubusercontent.com/15698722/81041937-a98a8700-8e74-11ea-99b9-297893caae4e.png)

**Comment by aaronhowser1** [comment]
![image](https://user-images.githubusercontent.com/15698722/81042111-02f2b600-8e75-11ea-8518-81b12504504e.png)
The old FTB Wiki says it uses *Item* Cards, which have no recipe. It also says it can use cards created from a Logistics Security Station, but those are a *lot* more broken
![image](https://user-images.githubusercontent.com/15698722/81042151-17cf4980-8e75-11ea-977a-778af3506471.png)

**Comment by bziemons** [comment]
Yeah, we got rid of the soldering station in 0.10 and the security station needs a bit of love. Not sure if we're going to use the satellite pipe logic in the future for connecting pipes. I think the linking cards are quite a hassle.

**Comment by aaronhowser1** [comment]
Should I avoid making quests for them then? Is there anything else I should avoid making quests for?

**Comment by bziemons** [comment]
Sounds like a good idea. Maybe avoid the fuzzy crafting altogether, because that wasn't maintained after it was kindly contributed and last time I tried to use it wasn't a success. There probably are quite a few more bugs than you already found. I am slowly refactoring code at the moment and trying to work on performance, so this could take quite a while..

**Comment by bziemons** [comment]
Btw. related: https://github.com/RS485/LogisticsPipes/issues/1159

**Comment by aaronhowser1** [comment]
Ah, so it's planned to be a bit more like ae2's p2p, except it still travels the whole path?

**Comment by bziemons** [comment]
dunno where that AE comparison came from, but I am going to quote from a comment on that:

> And this has nothing to do with AE2 channels. That doesn't even make sense in the context of entrance and destination pipes.

It is supposed to work like the satellite/crafting pipe link, like I said above.

**Comment by aaronhowser1** [comment]
I've not started playing with satellites yet. That guy was referencing AE2 because both had the word channel. I was referencing AE2 for a completely different reason. With AE2, you can place two P2P tunnels and have things enter one and exit the other, which is how I understand you're planning to change it to work (minus the teleporting).

**Comment by bziemons** [comment]
In that case: yes :) Sorry for the misunderstanding, I do not have a single clue about AE1 or 2

**Comment by aaronhowser1** [comment]
Awesome, that would be super useful. A *bit* of a shame that it was started in 2018 though, lol.

**Comment by bziemons** [comment]
Well, the 1.8 port which was the base for the 1.10 port and which is now part of this MC 1.12 version (first released last year) was started in 2016.. I don't mind if things take a while ;)

**Comment by bziemons** [comment]
Related to #1364
